src/chunk.c

```diff
@@ -3046,16 +3047,31 @@ ts_chunk_set_status(Chunk *chunk, int32 status)
 	return chunk_update_status(&chunk->fd);
 }
 
+/*
+ * Setting (INVALID_CHUNK_ID, true) is valid for an Access Node. It means
```

Comment:
did you mean to say is_compressed => true here?

Reply:
I meant (compressed_chunk_id => INVALID_CHUNK_ID, is_compressed => true)